dotCMS/src/integration-test/java/com/dotmarketing/util/TestConfig.java

```diff
@@ -0,0 +1,50 @@
+package com.dotmarketing.util;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class TestConfig {
+
+    public enum TestEnum {
+
+        ONE(1),
+        TWO(2),
+        THREE(3),
+        FOUR(4),
+        FIVE(5);
+
+        private int code;
+        private TestEnum(int code) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'code' is not assigned and could be declared final](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4502981)